### ISSUE TITLE
nan train loss while training SSD

### ISSUE THREAD

**Issue by sbhavani** [tryolabs/luminoth#277]
Getting train_loss: nan at every step running training for SSD. I tried varying batch_size and learning_rate but still no improvement.

InvalidArgumentError (see above for traceback): Nan in summary histogram for: ssd/ssd_feature_extractor/conv_4_3_norm/conv4_3_normalized_hist

**Comment by afaq-ahmad** [comment]
same problem
